config/autoload/global.php

```diff
@@ -13,7 +13,7 @@
     // will also disable it by default; clear the configuration cache using
     // `composer clear-config-cache`.
     ConfigAggregator::ENABLE_CACHE => true,
-    'base_url'                     => getenv('BASE_URL') ?? 'https://laminas.dev',
+    'base_url'                     => getenv('BASE_URL') ?: 'https://laminas.dev',
```

Comment:
There was a subtle change in either how null coalesce or getenv work between 7.4 and 8.1. Previously, `??` worked fine here. However, since getenv returns `false` on failure to lookup a value, `??` saw a non-null value, and would now use that. Switching to short ternary gets the intended behavior.